### PR TITLE
Fix Column handling

### LINKED ISSUE #1151: @Column annotation with no name property causes incorrect query to be created. Using JPASQLQuery.
For example I have a Company class which I want to select the name from (this is a contrived example but displays the bug):

```java
@Entity
public class Company {

  @Id
  @GeneratedValue
  private Long id;
  @Column(unique = true, nullable = false)
  private String name;
}
```
```java
JPASQLQuery sql = query.from(company).limit(10);
sql.list(company.name);
```
Will generate a query like
```sql
select company. from company limit ?
```

What I would have expected would be to use the field name as if the @Column annotation didn't exist in this case.

### PR DESCRIPTION
**Pull request by timowest**
Fixes #1151

### CODE REVIEW

File: querydsl-jpa/src/main/java/com/querydsl/jpa/NativeSQLSerializer.java [hunk 1]
```diff
@@ -57,7 +57,11 @@ protected void appendAsColumnName(Path<?> path, boolean precededByDot) {
         if (path.getAnnotatedElement().isAnnotationPresent(Column.class)) {
             SQLTemplates templates = getTemplates();
```
Comment (Shredder121) [inline querydsl-jpa/src/main/java/com/querydsl/jpa/NativeSQLSerializer.java]: These templates are only used in one branch of the if block, so you *could* move them in there.

**Comment by Shredder121** [comment]
I think that the fix is good, so I'll merge it, but I first want to hear your opinion on my suggestion.

**Comment by timowest** [comment]
Yes, your suggestion was valid and I made the change.

**Comment by Shredder121** [comment]
Then I'll merge them when the build completes.

**Pull request merged**